src/masoniteorm/orm/query/processors/SQLitePostProcessor.py

```diff
@@ -0,0 +1,28 @@
+class SQLitePostProcessor:
+    """Post processor classes are responsable for modifying the result after a query.
+
+    Post Processors are called after the connection calls the database in the
+    Query Builder but before the result is returned in that builder method.
+
+    We can use this oppurtunity to get things like the inserted ID.
+
+    For the Postgres Post Processor we have a RETURNING * string in the insert so the result
```

Comment:
@girardinsamuel Just like a inheritance to create a base and change just what will be different between "type of processors" ?

Reply:
I was just thinking about the docstrings 😛  but it could be applied to the real code too 👍 
Though for now it might not be necessary as there is not much there

Reply:
The doctoring yes I need to update lol. I copied and pasted the classes and these followed